code/modules/mob/living/carbon/superior_animal/life.dm

```diff
@@ -0,0 +1,86 @@
+/mob/living/carbon/superior_animal/Life()
+	. = ..()
+
+	objectsInView = null
+
+	if(client)
+		return .
```

Comment:
There's never any need to do "return ."

. contains the return value and will always be returned unless you specify "return somethingelse"